### ISSUE TITLE
Limit The BMP-2 / LAV-25

### ISSUE THREAD

**Issue by agentbaf** [ozon/b3-plugin-weaponlimiterbf3#7]
Hi, i want to limit the vehicle from being used in squad death match.

I currently have to have "Death" as weapon type set in the weaponlimiterbf3 config file as it doesn not seem to work when i have BMP-2 set as the weapon type in the config

2 problems i have from this are:
1.  the weapon "Death" includes mortors and possibly other weapons i havent discovered yet and not just the Tank.
2.  The warning text out puts as Warning: Weapon Death is forbidden, i cant see where i can customise this text so it at least says Tanks not death.

Is there any ways to limit just the vehicles in squad death match using this plugin so anyone that makes kills with it is slayed?

I hope this makes sense..?

**Comment by thomasleveil** [comment]
why don't you use the `!vehicule off` command from the poweradminbf3 plugin which prevent vehicules from spawning ?

For the "Death" weapon name, nothing can be done to get more precise info as this is the raw info B3 gets from the BF3 server.

**Comment by ozon** [comment]
Hi,
I wrote this Plugin for limit m320/RPG abuse on Metro and it do the Job.

Currently, I rewrite the Plugin. Some new features are coming. But I think the basic idea is to limit abuse weapons.
For vehicles is that much more complex and a '''!vehicule off''' works better. Do try to disable vehicles under certain conditions - **the poweradminbf3 does a great job. Disable Vehicle for the SQDM gametype** ( I do it too) . 

But I noticed the "weapons ban" only limited effect. For example, there is more sniper camping as RPG/m320 replacement.

Somewhere I had already written. that this plugin can not ensure a fair game.
